src/main/scala/inox/solvers/SolverFactory.scala

```diff
@@ -324,7 +367,9 @@ object SolverFactory {
     }
   }
 
-  def supportedSolver(s: String) = solverNames.contains(s) || s.startsWith("smt-z3:")
+  private def supportedSolverRaw(s: String) = solverNames.contains(s) || s.startsWith("smt-z3:")
+  def supportedSolver(s: String) =
```

Comment:
This isn't technically correct since only smt-z3, smt-cvc4 and unrollz3 allow no-inc